pineappl_py/tests/test_boc.py

```diff
@@ -59,9 +59,10 @@ def _generated_bwfl_fields(n_bins: int, n_dimensions: int) -> BwflFields:
 
 class TestChannel:
     def test_init(self):
-        le = Channel([([2, 2], 0.5)])
-        assert isinstance(le, Channel)
-        assert le.into_array() == [([2, 2], 0.5)]
+        channels = Channel([([2, 2], 0.5)])
```

Comment:
- could you add a second more complicated case with two entries? I'm not entirely sure what the function does ... 
- can you also clarify the name: should the variable be singular `channel` ? as in uubar channel here. I.e. the functions acts only _within_ a given channel, such as u-like-ubar-like (and not across to gluon-gluon)

Reply:
I made it a bit clearer in [dc22670](https://github.com/NNPDF/pineappl/pull/353/commits/dc22670cd3355211489b38baed6a6d15b0197659). Hope it is better now.

Here, the `Channel` object can contain multiple channel combinations. For instance, for a $q \bar{q}$-initiated channel, it'll contain all the $n_f$ active combinations. So in this sense it can be singular or plural.